Source/WavetableSynth.cpp

```diff
@@ -95,10 +96,7 @@ void WavetableSynth::render(juce::AudioBuffer<float>& buffer, int beginSample, i
         }
     }
 
-    for (int channel = 1; channel < buffer.getNumChannels(); ++channel)
-    {
-        auto* channelData = buffer.getWritePointer(channel);
-        std::copy(firstChannel + beginSample, firstChannel + endSample, channelData + beginSample);
-    }
+  
+        std::copy(firstChannel + beginSample, firstChannel + endSample, secondChannel + beginSample);

```

Comment:
Hi, thanks for the PR! :)
Why do you want to fix the synthesizer to play only in stereo? In this way, we cannot have mono or any of the more sophisticated configurations. What am I missing here?

Reply:
hi.
if i don't got it wrong you don't apply any stereo processing on your sinusoids. 
each channel has the same information and you don't need to have another for loop.
you just copy the left channel to the right channel. (index 0 and index 1)
if you want to have a stereo process you could make in your main audio thread where all the voices come together like a "master" effect. 

a different effect for each voice is a bit too much i think.
also you will need for each voice a different knob.
sinusoids have the least spectral information. 
a detuning effect or clipping could make them more interesting. 
(see shepard tones for detune, and by clipping you can get a simple squarewave by pushing it to the limit)
but then you will need like a GUI that the user could decide the amount of the effect and so on.

an amplitude envelope could also help to avoid clicks by fading the sound in and out.